prime-router/settings/STLTs/WA/wa-phd.yml

```diff
@@ -0,0 +1,119 @@
+              - "Bundle.entry.resource.ofType(Patient).where(address.state.lower() != 'unknown')"
+              # "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)"
+              - "(Bundle.entry.resource.ofType(Patient).address.line.empty().not() or Bundle.entry.resource.ofType(Patient).address.postalCode.empty().not() or Bundle.entry.resource.ofType(Patient).telecom.empty().not())"
+              # "hasAtLeastOneOf(order_test_date,specimen_collection_date_time,test_result_date)"
+              - "Bundle.entry.resource.ofType(DiagnosticReport).where( basedOn.resolve().extension('https://reportstream.cdc.gov/fhir/StructureDefinition/effective-date').empty() and specimen.resolve().collection.collected.empty() and effective.empty() and issued.empty() and result.resolve().effective.empty()).count() = 0"
+              # "hasValidDataFor(message_id,equipment_model_name,specimen_type,test_result,patient_last_name,patient_first_name,patient_dob,patient_city,patient_state)"
+              - "Bundle.identifier.value.empty().not()"
+              - "Bundle.entry.resource.ofType(DiagnosticReport).result.resolve().where(method.empty() or value.coding.code.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Patient).where(name.family.empty() or name.given.empty() or birthDate.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Patient).where(address.city.empty()).count() = 0"
+              - "Bundle.entry.resource.ofType(Patient).where(address.state.empty()).count() = 0"
+              # "isValidCLIA(testing_lab_clia,reporting_facility_clia)"
+              - "Bundle.entry.resource.ofType(ServiceRequest).performer.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
+              - "Bundle.entry.resource.ofType(ServiceRequest).requester.resolve().organization.resolve().identifier.where(type.coding.code = 'CLIA').value.getIdType() = 'CLIA'"
+      routingFilter:
+      #        # "matches(test_result_status, F, S, P, C, X, I)"
+              - "Bundle.entry.resource.ofType(DiagnosticReport).where(status in 'final'|'preliminary'|'amended'|'corrected').exists()"
+      processingModeFilter:
+      # "doesNotMatch(processing_mode_code,T,D)"
+              - "Bundle.entry.resource.ofType(MessageHeader).meta.tag.where(system = 'http://terminology.hl7.org/CodeSystem/v2-0103').code.exists() and Bundle.entry.resource.ofType(MessageHeader).meta.tag.where(system = 'http://terminology.hl7.org/CodeSystem/v2-0103').code != 'T' and Bundle.entry.resource.ofType(MessageHeader).meta.tag.where(system = 'http://terminology.hl7.org/CodeSystem/v2-0103').code != 'D'"
+      reverseTheQualityFilter: false
+      conditionFilter:
+        # "matches(abnormal_flag, A)"
+        - "Bundle.entry.resource.ofType(Observation).interpretation.coding.code = 'A'"
+        # Accept COVID only
```

Comment:
The interpretation code needs to be integrated into the condition filter below.  Doing it this way may filter out all the observations in the bundle.

```
- "(%resource.interpretation.coding.code = 'A' and %resource.code.coding.extension('https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code').value.where(code in ('840539006')).exists())"
```

Reply:
Got it, thank you